Code/Framework/AzFramework/AzFramework/Spawnable/Script/SpawnableScriptMediator.cpp

```diff
@@ -161,6 +163,8 @@ namespace AzFramework::Scripts
             return false;
         }
 
+        m_activeSpawnTickets.erase(spawnTicket);
```

Comment:
I believe it is possible to use the same ticket for spawning and despawning multiple prefabs. So, this shouldn't be done. If there was a 'destroy ticket' node added in the future, we should do this then. But we can probably add that later based on user feedback.